app/services/neighbour_response_creation_service.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class NeighbourResponseCreationService
+  class CreateError < StandardError; end
+
+  def initialize(planning_application:, **options)
+    @planning_application = planning_application
+    options.each { |k, v| instance_variable_set("@#{k}", v) unless v.nil? }
```

Comment:
I know I did something similar with the `PlanningApplicationCreationService` but I think we are only ever passing `params` and `planning_application` to initialize this service right?